### PR TITLE
[Docs] Clarify get-events API usage

### PR DESCRIPTION
**Pull request by seanshi-scale**
Also update docs to reflect that mistral-7b is available

### CODE REVIEW

**Review by saiatmakuri — APPROVED** [review]

**Comment by yixu34** [comment]
Don't merge yet, was going to make some comments

File: docs/guides/fine_tuning.md [hunk 1]
```diff
@@ -152,7 +152,15 @@ See the [Model Zoo](../../model_zoo) to see which models have fine-tuning suppor
 
 See [Integrations](../integrations.md) to see how to track fine-tuning metrics.
 
-Once the fine-tune is launched, you can also [get the status of your fine-tune](../../api/python_client/#llmengine.fine_tuning.FineTune.get). You can also [list events that your fine-tune produces](../../api/python_client/#llmengine.fine_tuning.FineTune.get_events).
+## Monitoring the fine-tune
+
+Once the fine-tune is launched, you can also [get the status of your fine-tune](../../api/python_client/#llmengine.fine_tuning.FineTune.get).
```
Comment (yixu34) [inline docs/guides/fine_tuning.md:157]: This is great, but can we sprinkle code snippets for examples on how to do various things in here?

Would also be good to have snippets of error examples. Basically we're trying to make this more tangible with examples.

File: docs/guides/fine_tuning.md [hunk 2]
```diff
@@ -152,7 +152,35 @@ See the [Model Zoo](../../model_zoo) to see which models have fine-tuning suppor
...
+
+
+```
+
+The status of your fine-tune will give a high-level overview of the fine-tune's progress.
+The events of your fine-tune will give more detail, such as the training loss and validation loss at each epoch, 
+as well as any errors that may have occurred. If you encounter any errors with your fine-tune, 
+the events are a good place to start debugging. For example, if you see `Unable to read training or validation dataset`,
+you may need to make your files accessible to LLM Engine. If you see `Invalid value received for lora parameter 'lora_alpha'!`,
+you should check that your hyperparameters are valid.
```
Comment (yixu34) [inline docs/guides/fine_tuning.md:183]: Might be good to add a reminder link to the canonical definition of valid hyperparameters?

File: docs/guides/fine_tuning.md [hunk 3]
```diff
@@ -152,7 +152,35 @@ See the [Model Zoo](../../model_zoo) to see which models have fine-tuning suppor
...
+# timestamp=1697590000.0 message="{'eval_loss': 23.456, 'eval_runtime': 19.876, 'eval_samples_per_second': 4.9, 'eval_steps_per_second': 4.9, 'epoch': 0.97}" level='info'
+# timestamp=1697590020.0 message="{'train_runtime': 421.234, 'train_samples_per_second': 2.042, 'train_steps_per_second': 0.042, 'total_flos': 123.45, 'train_loss': 34.567, 'epoch': 0.97}" level='info'
+
+
+```
+
+The status of your fine-tune will give a high-level overview of the fine-tune's progress.
+The events of your fine-tune will give more detail, such as the training loss and validation loss at each epoch, 
+as well as any errors that may have occurred. If you encounter any errors with your fine-tune, 
+the events are a good place to start debugging. For example, if you see `Unable to read training or validation dataset`,
```
Comment (yixu34) [inline docs/guides/fine_tuning.md:181]: Do we have a separate error for malformed files, e.g. files that don't have `prompt,response` headers?

Would it make sense to put these in a table?

Comment (yixu34) [reply docs/guides/fine_tuning.md:181]: Discussed offline: `prompt,response` header validation would happen earlier, at `FineTune.create` time.

**Review by yixu34 — APPROVED** [review]
